app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -276,12 +280,10 @@ private void enableBluetooth(final boolean buttonClicked) {
             final Intent intent = new Intent(this, BluetoothConnectionActivity.class);
             this.startActivity(intent);
         } else if (!bluetoothAdapter.isEnabled()) {
-            final String description;
-            if (buttonClicked) {
-                description = this.getString(R.string.popup_enable_bluetooth_exchange);
-            } else {
-                description = this.getString(R.string.popup_enable_bluetooth_findable);
-            }
+            final String description = this.getString(
+                    buttonClicked
+                            ? R.string.popup_enable_bluetooth_exchange
+                            : R.string.popup_enable_bluetooth_findable);
```

Comment:
This formatting looks awful to me 😆 , not the ternary operator but the newlines btw